src/main/webapp/assets/js/config.js

```diff
@@ -100,3 +100,17 @@ require([
 ], function (Router) {
     new Router().startBrooklynGui();
 });
+
+/*
+ * Prepend a base URL to REST API calls
+ */
+$.ajaxSetup({
+    beforeSend: function(jqXHR, settings) {
+//        var baseURL = "/api/brooklyn/";
+        var baseURL = "";
+
+        if (baseURL && settings.url.match(/^\/v1\/\w+/i)){
```

Comment:
True it would be more clear, but the way it's currently set up is to intercept only calls to URLs that begin with **/v1/**, so something like **/v1/projects** will be affected, while **/api/v1/projects** won't.

If you use `.indexOf('/v1/') == 0`, **/api/v1/projects** will be affected as well.